src/pytest_plugins/filler/filler.py

```diff
@@ -132,7 +132,7 @@ def pytest_addoption(parser: pytest.Parser):
         action="store_true",
         dest="flat_output",
         default=False,
-        help="Output each test case in the directory without the folder structure.",
+        help="[DEPRECATED] Output each test case in the directory without the folder structure. This flag will be removed in a future version.",
```

Comment:
```suggestion
        help="[DEPRECATED] Output each test case in the directory without the folder structure. "
        "This flag will be removed in a future version.",
```
Exceeds the character limit so the linter is going to complain.